airbyte-cdk/python/setup.py

```diff
@@ -35,7 +35,7 @@
 
 setup(
     name="airbyte-cdk",
-    version="0.1.8",
+    version="0.1.10",
```

Comment:
you'll need to either merge this with arthur's changes or release separately

Reply:
Ive merged Arthur's branch into mine so now this PR looks dirty before his branch will be merged on master.